azure-pipelines.yml

```diff
@@ -417,38 +417,38 @@ stages:
           displayName: 'Run ITs'
           strategy:
             matrix:
-              vs2017_latest89:
+              vs2017_latest99:
                 PRODUCT: "SonarQube"
-                SQ_VERSION: "LATEST_RELEASE[8.9]"
-                SONAR_CFAMILYPLUGIN_VERSION: "6.20.5.49286"  # LATEST_RELEASE of CFAMILY is not compatible with old SQ
+                SQ_VERSION: "LATEST_RELEASE[9.9]"
+                SONAR_CFAMILYPLUGIN_VERSION: "6.44.0.61773"  # LATEST_RELEASE of CFAMILY might not be compatible with SQ
                 SONAR_CSHARPPLUGIN_VERSION: "DEV"
                 SONAR_VBNETPLUGIN_VERSION: "DEV"
                 MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\MSBuild\\15.0\\Bin\\MSBuild.exe"
                 PLATFORMTOOLSET: "v140"
                 WINDOWSSDKTARGET: "10.0.17763.0"
-                JDKVERSION: "1.11"
+                JDKVERSION: "1.17"
                 TEST_SUITE: "**/SonarQubeTestSuite.java"
-              vs2019_latest79:
+              vs2019_latest99:
                 PRODUCT: "SonarQube"
                 SQ_VERSION: "LATEST_RELEASE[7.9]"
-                SONAR_CFAMILYPLUGIN_VERSION: "6.3.0.11371"  # LATEST_RELEASE of CFAMILY is not compatible with old SQ
+                SONAR_CFAMILYPLUGIN_VERSION: "6.44.0.61773"  # LATEST_RELEASE of CFAMILY might not be compatible with SQ
                 SONAR_CSHARPPLUGIN_VERSION: "DEV"
                 SONAR_VBNETPLUGIN_VERSION: "DEV"
                 MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\MSBuild\\Current\\Bin\\MSBuild.exe"
                 PLATFORMTOOLSET: "v140"
                 WINDOWSSDKTARGET: "10.0.17763.0"
-                JDKVERSION: "1.11"
+                JDKVERSION: "1.17"
                 TEST_SUITE: "**/SonarQubeTestSuite.java"
-              vs2022_latest89:
+              vs2022_latest99:
```

Comment:
The purpose here was to add VS2022 with SQ 9.9. Without changing the existing tests.

We still need to preserve ITs with 7.9 and 8.9, until we drop support in v6.x

Reply:
And this was just a comment to help you, not a review :)

Reply:
For analyzers, we need only the latest LTS, without the previous ones.
For Scanner, we need more of them.